sale_order_product_recommendation/views/sale_order_view.xml

```diff
@@ -1,22 +1,18 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<!-- Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
```

Comment:
Don't remove copyright here. You may add yours though.

Reply:
done

Reply:
@cesar-tecnativa would you like to add your copyright here?